app/views/browse_everything/_modal.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="modal fade" id="browse-everything-modal" tabindex="-1" role="dialog" aria-labelledby="browse-everything-modal-title" aria-hidden="true">
```

Comment:
This overrides something from the browse everything gem? It would be nice to have a comment with a permanent link to the overridden file and a reason why we override it.